heron/proto/ckptmgr.proto

```diff
@@ -0,0 +1,197 @@
+package heron.proto.ckptmgr;
+option java_package = "com.twitter.heron.proto.ckptmgr";
+option java_outer_classname = "CheckpointManager";
+
+import "common.proto";
+import "physical_plan.proto";
+
+// message stored in the state manager by the tmaster
+message StatefulConsistentCheckpoints {
+  message StatefulConsistentCheckpoint {
```

Comment:
Why an inner class? If we make this a top level class it can be used on its own without the collection, like in the RPC for example while working with a specific checkpoint.